frontend/src/components/Smart/index.jsx

```diff
@@ -8,9 +8,11 @@ import AdminTableContainer from '../../containers/adminTable_container';
 import RecycleBinContainer from '../../containers/recycleBin_container';
 import CodebookLabelMenuContainer from '../../containers/codebookLabelMenu_container';
 import SmartProgressBarContainer from '../../containers/smartProgressBar_container';
+import UnlabeledDataStatusContainer from '../../containers/unlabeled_data_status_container';
 
-const ADMIN = window.ADMIN;
 
+const ADMIN = window.ADMIN;
+const PROJECT_ID = window.PROJECT_PK;
```

Comment:
The linter finds the following error with this line:

```
/code/src/components/Smart/index.jsx
  15:7  warning  'PROJECT_ID' is assigned a value but never used  no-unused-vars
```

To recreate the error, run the linting command `docker-compose -f envs/dev/docker-compose.yml run --rm smart_frontend yarn lint`